src/api/operations/project.queries.graphql

```diff
@@ -167,6 +167,12 @@ query GetProjectReferralPostings($id: ID!, $limit: Int = 10, $offset: Int = 0) {
   }
 }
 
+query GetReferralPosting($id: ID!) {
```

Comment:
Can we move the referral-related queries and fragments into their own files? I see we already have `referralRequest.queries`, maybe that could be renamed to `referrals` and include everything referral-related.